src/calculator.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import numbersAndSymbols from "./arrayData"
+
+let arrayNumber = []
+let arrayNumberAndSigns = [] //este se puede usar en vez del otro, revisar 
+let newArrayStrings = []
```

Comment:
declare this inside the class, as a property https://www.freecodecamp.org/news/javascript-classes-how-they-work-with-use-case/